### ISSUE TITLE
[Feature request] Activity history on profiles

### ISSUE THREAD

**Issue by Arrow420** [zend10/AL-chan#53]
You should add the activity history thing to profiles, I think it's called 'UserActivityHistory' in the API. Under the bio could be good place for it.

Photo from Anilist:
![](https://i.imgur.com/qrhScH3.png)

The API:
![](https://i.imgur.com/JimOHUr.png)

**Comment by zend10** [comment]
Yeah been eyeing this. Would be great if there's a library for this component that I can use right away.
